compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -3211,6 +3211,101 @@ TR_S390Peephole::AGIReduction()
 
    return performed;
    }
+
+/* For runs with -Xgc:concurrentScavenge on hardware that doesn't support guarded storage facility,
```

Comment:
Please use Doxygen syntax for the comment.

Reply:
fixed the comment